### PR TITLE
[dash] Cleaned up the oem page markdown, and added remaining Inside Flutter text

### PR DESCRIPTION
**Pull request by Sfshaza**
@Hixie, please review.

Note that the last several footnotes aren't linked to. I couldn't find the links in the source doc, either.

Fixed some minor misspellings here and in the source doc.

### CODE REVIEW

File: src/resources/inside-flutter.md [hunk 1]
```diff
@@ -328,6 +331,258 @@ for efficiently updating elements during scrolling and for modifying
 the render tree when elements are scrolled into and out of view at
 the edge of the viewport.
 
+# API Ergonomics
```
Comment (Hixie) [inline src/resources/inside-flutter.md]: The earlier section uses \` for class names, but this section doesn't (e.g. "Widget", "Element"). Is that intentional?

File: src/testing/oem-debuggers.md [hunk 2]
```diff
@@ -13,23 +13,19 @@ that portion of your code using XCode (for iOS) or Android Gradle (for Android).
 This guide shows you how you can connect _two_ debuggers to your Dart app,
```
Comment (Hixie) [inline src/testing/oem-debuggers.md]: didn't review the edits to this file

**Comment by Hixie** [comment]
LGTM

**Comment by Hixie** [comment]
(Not sure what you mean by the last footnotes not being linked to?)

File: src/testing/oem-debuggers.md (hunk 2 above)
Comment (Sfshaza) [reply src/testing/oem-debuggers.md]: That's fine. Jacob R has reviewed.

File: src/resources/inside-flutter.md (hunk 1 above)
Comment (Sfshaza) [reply src/resources/inside-flutter.md]: No, I'll fix. It's a misunderstanding between me and our contractor.

**Comment by Sfshaza** [comment]
Footnotes a5 through a8 aren't referenced from the text. That's all.

**Pull request merged**